### PR TITLE
Minor changes in documentation

### PR DESCRIPTION
**Pull request by soosub**
## Context for changes

Small changes and fixes to documentation.

- **Documentation changes**:
    * Fix spelling MN
    * Disambiguate CV and DV in the README.

## Example usage and tests
- [x] Not Applicable

## Performance results justifying changes
- [x] Not Applicable

## Workflow actions and tests
- [x] Not Applicable

## Checklist and integration statements

- [x] My Python and C++ codes follow this project's coding and commenting styles as indicated by existing files. Precisely, the changes conform to given `black`, `docformatter` and `pylint` configurations. 
- [x] I have performed a self-review of these changes, checked my code (including for [codefactor](https://www.codefactor.io/repository/github/xanaduai/flamingpy/branches) compliance), and corrected misspellings to the best of my capacity. I have synced this branch with others as required.

- [x] I have added context for corresponding changes in documentation and [`README.md`](README.md) as needed.
- [x] I have added new workflow CI tests for corresponding changes, ensuring codecoverage is 95% or better, and these pass locally for me.
- [x] I have updated [`CHANGELOG.md`](.github/CHANGELOG.md) following the template. I recognize that the developers may revisit [`CHANGELOG.md`](.github/CHANGELOG.md) and the versioning, and create a Special Release including my changes.

### CODE REVIEW

**Comment by soosub** [comment]
@ilan-tz I'd prefer to tackle

> Make sure non-Xanadu links open in a new page (this will require some clicking around to see when this happens)

sometime later. I'd propose to merge this one now (so the docs don't contain spelling mistakes) and complete the other task in a later PR.

File: README.md [hunk 1]
```diff
@@ -91,19 +91,19 @@ from flamingpy.cv.ops import CVLayer
 from flamingpy.decoders import correct
 ```
 
-Next, let us instantiate an RHG lattice -- the measurement-based version of the surface code:
+Next, let us instantiate an RHG lattice -- the measurement-based version of the surface code that we will use as our DV layer:
```
Comment (ilan-tz) [inline README.md:94]: Let's please remove the changes below, and just keep this one!

File: flamingpy/_version.py [hunk 2]
```diff
@@ -15,4 +15,4 @@
 """Version number (major.minor.patch[label])"""
 
 
-__version__ = "0.8.2a5-dev2"
+__version__ = "0.8.2a5.dev3"
```
Comment (soosub) [inline flamingpy/_version.py:18]: @ilan-tz is there a reason why this is a dot `.` and the other is a hyphen `-`?

Comment (ilan-tz) [reply flamingpy/_version.py:18]: @nariman87 mentioned the dot is more pythonic (and indeed matches [PEP 440](https://peps.python.org/pep-0440/). We'll use it from now on!

Comment (soosub) [inline flamingpy/_version.py:18]:
```suggestion
__version__ = "0.8.2a5-dev3"
```

**Review by soosub — COMMENTED** [review]
for your convenience

File: flamingpy/_version.py (hunk 2 above)
Comment (soosub) [reply flamingpy/_version.py:18]: Ah I see, in that case you can ignore my suggestion and merge it I think

**Review by ilan-tz — APPROVED** [review]

**Pull request merged**